l10n_it_fatturapa_out/tests/test_fatturapa_xml_validation.py

```diff
@@ -715,6 +753,7 @@ def test_multicompany_fail(self):
         with invoice1_form.line_ids.new() as line_form:
             line_form.product_id = self.product_product_10
             line_form.account_id = self.a_sale
+            line_form.tax_ids.clear()
```

Comment:
come mai aggiungi questa riga?

Reply:
Più per simmetria che altro, si presuppone che la lista sia vuota in quel punto, con la clear() me ne assicuro. Nell'altro test mancava e lo ho aggiunto.